lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -27,9 +27,21 @@ class AwsCompileCloudFrontEvents {
       'before:remove:remove': this.logRemoveReminder.bind(this),
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8025
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'cloudFront', {
       type: 'object',
+      properties: {
+        eventType: {
+          type: 'string',
+          enum: ['viewer-request', 'origin-request', 'origin-response', 'viewer-response'],
+        },
+        pathPattern: { type: 'string', pattern: '^([A-Za-z0-9-_.*$/~"\'@:+]|&amp;)+$' },
```

Comment:
I think doc mention that `&` is accepted and that it'll translate to `&amp;`. If I understand correctly `&amp;` should not be part of expression

Reply:
The doc says:
> &, passed and returned as &amp;
I understood from that: if you want to include & in path pattern, pass it as &amp

Reply:
Indeed, I didn't note coma :)